scrape/scrape.go

```diff
@@ -445,11 +454,11 @@ func (sp *scrapePool) Sync(tgs []*targetgroup.Group) {
 	var all []*Target
 	sp.droppedTargets = []*Target{}
 	for _, tg := range tgs {
-		targets, err := targetsFromGroup(tg, sp.config)
-		if err != nil {
+		targets, failures := targetsFromGroup(tg, sp.config)
+		for _, err := range failures {
 			level.Error(sp.logger).Log("msg", "creating targets failed", "err", err)
```

Comment:
```suggestion
			level.Error(sp.logger).Log("msg", "Creating target failed", "err", err)
```

This is now for just one target.

Reply:
Whoops. Nice catch!